cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/util/AbstractAppFabricHttpHandler.java

```diff
@@ -136,7 +136,7 @@ protected ProgramType getProgramType(String programType) {
   protected void getLiveInfo(HttpResponder responder, Id.Program programId,
                              ProgramRuntimeService runtimeService) {
     try {
-      responder.sendJson(HttpResponseStatus.OK, runtimeService.getLiveInfo(programId));
+      responder.sendJson(HttpResponseStatus.OK, runtimeService.getLiveInfo(programId.toEntityId()));
```

Comment:
who calls this method? If it is only called in ``ProgramLifecycleHttpHandler``,  better move it there, and accept ``ProgramId`` instead of ``Id.Program``?